### DISCUSSION TITLE
Problems encountered by beginners using ESP32

### DISCUSSION THREAD

**Discussion by lzxmjy** [community/community#120259] [Discussions]
### Select Topic Area

Bug

### Body

The computer connected to ESP32 CH340 can detect serial port 5, but the following error occurs

esptool.py v4.5.1
Serial port COM5
Connecting......................................

A fatal error occurred: Failed to connect to ESP32: No serial data received.

exit status 2

**Comment by LiteBrite82** [comment]
Hi @lzxmjy ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this topic. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!
